### PR TITLE
revise help text for addinvoice lncli command

### PR DESCRIPTION
**Pull request by st4rgut24**
## Change Description
Revise help text for addinvoice lncli command

### Code Style and Documentation
- [x] The change obeys the [Code Documentation and Commenting](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#CodeDocumentation) guidelines, and lines wrap at 80.
- [ ] Commits follow the [Ideal Git Commit Structure](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#IdealGitCommitStructure).
- [ ] Any new logging statements use an appropriate subsystem and logging level.
- [ ]  [There is a change description in the release notes](https://github.com/lightningnetwork/lnd/tree/master/docs/release-notes), or `[skip ci]` in the commit message for small changes.

📝 Please see our  [Contribution Guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md) for further guidance.

### CODE REVIEW

**Review by yyforyongyu — CHANGES_REQUESTED** [review]
Thanks for the PR! According to our contribution guidelines,
> [There is a change description in the release notes](https://github.com/lightningnetwork/lnd/tree/master/docs/release-notes), or [skip ci] in the commit message for small changes.

could you change your commit message to `cli: revise help text for addinvoice lncli command [skip ci]`

File: cmd/lncli/cmd_invoice.go [hunk 1]
```diff
@@ -18,7 +18,7 @@ var addInvoiceCommand = cli.Command{
 
 	Invoices without an amount can be created by not supplying any
 	parameters or providing an amount of 0. These invoices allow the payee
-	to specify the amount of satoshis they wish to send.`,
+	to specify the amount of satoshis they wish to receive.`,
```
Comment (guggero) [inline cmd/lncli/cmd_invoice.go:21]: I think this is the wrong way around. If the payee wants to specify the number of satoshis to receive, they use the `--amt` field. If they want to give the option to the other side (the **payer**), then they leave the amount as zero. So this should say "allows the payer ... they with to send".

**Review by guggero — CHANGES_REQUESTED** [review]

**Comment by yyforyongyu** [comment]
Thanks for the update. I think you need to squash the two commits and then we'd be good to go.

**Comment by st4rgut24** [comment]
@yyforyongyu , @guggero  ready for a final review

**Review by guggero — APPROVED** [review]

**Review by yyforyongyu — APPROVED** [review]

**Pull request merged**